lib/usb-connection.js

```diff
@@ -23,7 +29,7 @@ try {
   // var VENDOR_REQ_IN  = usb.LIBUSB_REQUEST_TYPE_VENDOR | usb.LIBUSB_RECIPIENT_DEVICE | usb.LIBUSB_ENDPOINT_IN;
 } catch (e) {
   haveusb = false;
-  logs.err('WARNING: No usb controller found on this system.');
+  log.error('WARNING: No usb controller found on this system.');
 }
```

Comment:
I ran into this error while testing this patch - `[Error: Error: Module version mismatch. Expected 47, got 46.]`, which came from the internals of the `usb` module and lefts the process hanging until I force quit it with `kill -9`. That error didn't come through until I added `log.error(e)` above line 31. I'm still not sure why it was left hanging open though because this didn't happen on `master`. Maybe there should be a `Promise.reject()`? 

The solution for that error was to reinstall my `node_modules` by the way.